### PR TITLE
Set up front end for reg commenting

### PR DESCRIPTION
**Pull request by Scotchester**
Short description explaining the high-level reason for the pull request

## Additions

- reg-comment organism HTML and CSS
- reg-comment organism made available to main content area on sublanding pages

## Removals

- Reg comment form test page

## Changes

- Updates `@btn-font-size` variable to match Design Manual standard of 16px

## Testing

1. Pull this fork down
2. **If you didn't test #1194:**
 - Copy the new env vars to your `.env` file
 - Replace `<api_key>` with the actual API key
 - `cd` out and back into `cfgov-refresh` to activate the new env vars
 - `pip install -r requirements/local.txt`
3. `./cfgov/manage.py migrate`
4. `./cfgov/manage.py runserver`
5. Log into Wagtail running locally
6. Create a new sublanding page; give it a title
7. Add the Reg Comment module to the content area
8. Enter the test Document ID: `CFPB_FRDOC_0001-0439`
9. Publish to WWW
10. View the page, test the form
11. See result page 🎊 

## Review

- @kurtw 
- @rosskarchner 
- @richaagarwal 
- @kave 
- @sebworks 
- @amymok 

## Screenshots
screen shot 2016-04-25 at 15 22 40

## Todo (not in this PR but after launch)
- Add a deadline for comments to the Wagtail module, set the form to close automatically after the deadline.

## Checklist

* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

**Comment by kurtw** [comment]
Functional 👍 but I'm not picasso

File: cfgov/jinja2/v1/_includes/organisms/reg-comment.html [hunk 1]
```diff
@@ -1,50 +1,95 @@
...
+        <div class="m-form-field-with-button">
+            <div class="form-group">
+                <label for="first_name">
+                    First name
+                </label>
+                <input type="text" id="first_name" name="first_name" required>
+            </div>
+        </div>
+
+        <div class="m-form-field-with-button">
```
Comment (sebworks) [inline cfgov/jinja2/v1/_includes/organisms/reg-comment.html]: Is there a reason that you added `m-form-field-with-button`?

File: cfgov/jinja2/v1/_includes/organisms/reg-comment.html [hunk 2]
```diff
@@ -1,50 +1,95 @@
...
+            </div>
+        </div>
 
+        <div class="m-form-field-with-button">
+            <div class="form-group">
+                <label for="email">
+                    Email address
+                    <span class="label_note">(optional)</span>
+                </label>
+                <input type="text" id="email" name="email">
```
Comment (sebworks) [inline cfgov/jinja2/v1/_includes/organisms/reg-comment.html]: input type should be `email` and you probably want placeholder text with the value set to `example.com`

**Comment by sebworks** [comment]
👍 , minor comments. Not sure if you want to use JS to validate the form and display a notification.

File: cfgov/jinja2/v1/_includes/organisms/reg-comment.html (hunk 1 above)
Comment (Scotchester) [inline cfgov/jinja2/v1/_includes/organisms/reg-comment.html]: I was trying to understand the atomic structure and follow it as best as I could, but I think ultimately it doesn't end up doing anything here. Perhaps I'll remove it so future changes to that molecule don't inadvertently affect these forms.

File: cfgov/jinja2/v1/_includes/organisms/reg-comment.html (hunk 2 above)
Comment (Scotchester) [inline cfgov/jinja2/v1/_includes/organisms/reg-comment.html]: Good call.

**Comment by Scotchester** [comment]
Fixes pushed to address comments by @sebworks. JS validation is out of scope for this initial release, but would like to do that eventually.

**Pull request merged**